### PR TITLE
Add workflow for RV2

### PR DESCRIPTION
**Pull request by JonatanAntoni**

### CODE REVIEW

**Comment by VladimirUmek** [comment]
I just saw this now - unfortunately RTOS2_Validation is not functional at the moment, failures start with build due to various reasons (the way packs are installed, RTX requires OS Tick:SysTick). We need to fix that first...

File: .github/workflows/cmsis_rv2.yml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+          vcpkg x-update-registry --all
+          vcpkg activate
+          
+          echo "Fix library search path for VHT models"
+          echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$(dirname $(which VHT_MPS2_Cortex-M3))" >> $GITHUB_ENV
+
+          echo "Fix execute permissions for VHT models"
+          chmod +x $(dirname $(which VHT_MPS2_Cortex-M3))/VHT_*
+
+          echo "Patch CMSIS-Toolbox"
```
Comment (VladimirUmek) [inline .github/workflows/cmsis_rv2.yml:95]: These patches are not necessary since cmsis-toolbox 2.10 was released

Comment (JonatanAntoni) [reply .github/workflows/cmsis_rv2.yml:95]: Yes, this PR is slightly outdated and needs updates anyway. We can revive it once issues are solved.

Comment (JonatanAntoni) [reply .github/workflows/cmsis_rv2.yml:100]: Removed

File: .github/workflows/cmsis_rv2.yml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      - name: Prepare vcpkg env
+        working-directory: ./CMSIS-RTOS2_Validation/Project
+        run: |  
+          . <(curl https://aka.ms/vcpkg-init.sh -L)
+          vcpkg x-update-registry --all
+          vcpkg activate
+          
+          echo "Patch CMSIS-Toolbox"
+          pushd $(dirname $(which cbuild))/../etc
+          curl -O https://raw.githubusercontent.com/Open-CMSIS-Pack/devtools/main/tools/buildmgr/cbuildgen/config/CLANG.17.0.1.cmake
```
[the remark this answers is not in the document]
Comment (JonatanAntoni) [reply .github/workflows/cmsis_rv2.yml:88]: True, let me update the env.

**Pull request merged**